softgroup/basis/common-router/router-impl/src/main/java/com/softgroup/first/router/configuration/FirstRouterAppCfg.java

```diff
@@ -1,4 +1,4 @@
-package com.softgroup.main.router.configuration;
+package com.softgroup.first.router.configuration;
```

Comment:
@AlexKazmirchuk you moved this to package router-impl, but package is still com.softgroup.first.router